### DISCUSSION TITLE
Using Google Maps on iOS causes poor Animation performance

### DISCUSSION THREAD

**Discussion by centerfield77** [react-native-maps/react-native-maps#4713] [Q&A]
"react-native": "0.69.6",
"react-native-maps": "0.31.0",

---

The app I am currently developing moves markers frequently on the map.

https://github.com/react-native-maps/react-native-maps#animated-marker-position
It is moved using 'timing' as described here.

It works fine when using Apple maps.
but, as soon as I switch to Google Maps, performance drops and the animation is choppy.

Does anyone have the same experience?
And does anyone know of a solution?

**Comment by stjasink** [comment]
The version of react-native-maps that you are using is a year old.  Have you tried with the latest version?  It might not fix it, but worth trying at least.

**Comment by harlandjp** [comment]
@stjasink 
I'm having the same issue with the following combo:
- "react-native-maps": "1.7.1",
- "react-native-reanimated": "~2.14.4",
- "react-native": "0.69.6",
- "expo": "^46.0.16",

Marker animation works fine with Apple Maps, but with Google Maps it's choppy or sometimes doesn't animate all all.

I implemented the approach suggested in https://github.com/react-native-maps/react-native-maps/issues/4371#issuecomment-1344148890 But no luck.

Any pointers would be highly appreciated 🙏
